corehq/apps/export/tests/test_export_views.py

```diff
@@ -151,6 +151,7 @@ def test_create_form_export(self):
         )
         self.assertEqual(resp.status_code, 200)
 
+    @patch("corehq.apps.export.views.new.get_case_types_for_domain", lambda *a: ['random_case'])
```

Comment:
Seems like it would be easy enough to add a test that verifies the behavior when `get_case_types_for_domain` doesn't return the passed in `case_type` right?

Reply:
I think so. I can add that test. Thanks.

Reply:
https://github.com/dimagi/commcare-hq/pull/35035/commits/b744b1e60783b89496086d939c47d58999f6aed8